CHANGES_UNRELEASED.md

```diff
@@ -2,10 +2,4 @@
 
 # Unreleased Changes
 
-## Nimbus SDK
```

Comment:
Shouldn't these changes end up in CHANGELOG.md?

Reply:
@mhammond  is this correct now?